app/pages/lab-pages-editor/components/WorkflowSettingsPage.jsx

```diff
@@ -47,7 +47,12 @@ export default function WorkflowSettingsPage() {
   if (!workflow) return null;
 
   return (
-    <form className="workflow-settings-page" onSubmit={onSubmit}>
+    <form
+      aria-labelledby="pages-editor_workflow-header-tab-button_settings"
```

Comment:
These IDs need to be maintained in a couple of different places, which makes the code brittle. You can make the component more flexible by passing IDs as component props.